### DISCUSSION TITLE
RemoveMicrosoftApplicationInsightsLoggerProvider() is used in inProcess but not in isolated

### DISCUSSION THREAD

**Discussion by fgheysels** [arcus-azure/arcus.templates#783] [Q&A]
Why do we use the `RemoveMicrosoftApplicationInsightsLoggerProvider()` method in InProcess function templates, and not in isolated function templates ?

**Comment by stijnmoreels** [comment]
Because we have more control over het logging functionality in isolated environments. There, you are responsible for all infrastructure, while in-process functions already have built-in infrastructure (which we can't remove by default).
So, we need it in in-process environments and not in isolated ones.
